#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_amd64_gcc40_fix.dpatch by Florian Ernst <florian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: To build on amd64 with gcc-4.0, apparently not needed anymore as of 2.40.

@DPATCH@
diff -urNad blender-2.40~/source/blender/python/api2_2x/CurNurb.c blender-2.40/source/blender/python/api2_2x/CurNurb.c
--- blender-2.40~/source/blender/python/api2_2x/CurNurb.c	2005-12-01 14:30:43.000000000 +0100
+++ blender-2.40/source/blender/python/api2_2x/CurNurb.c	2006-01-09 10:30:21.000000000 +0100
@@ -105,7 +105,7 @@
  * these methods are invoked on an instance of the type.
 */
 
-static PyMethodDef BPy_CurNurb_methods[] = {
+PyMethodDef BPy_CurNurb_methods[] = {
 /*   name,     method,                    flags,         doc               */
 /*  {"method", (PyCFunction) CurNurb_method, METH_NOARGS, " () - doc string"} */
 	{"setMatIndex", ( PyCFunction ) CurNurb_setMatIndex, METH_VARARGS,
